meta-balena-common/recipes-connectivity/libmbim/libmbim_1.24.2.bb

```diff
@@ -7,10 +7,11 @@ LIC_FILES_CHKSUM = " \
     file://COPYING.LIB;md5=4fbd65380cdd255951079008b364516c \
 "
 
-DEPENDS = "glib-2.0 glib-2.0-native libgudev autoconf-archive-native"
```

Comment:
This removal seems to break the build - https://jenkins.dev.resin.io/job/yocto-ts4900/3394/console

```
| autoreconf: running: /yocto/resin-board/build/tmp/work/cortexa9hf-neon-poky-linux-gnueabi/libmbim/1.24.2-r0/recipe-sysroot-native/usr/bin/autoconf --include=/yocto/resin-board/build/tmp/work/cortexa9hf-neon-poky-linux-gnueabi/libmbim/1.24.2-r0/libmbim-1.24.2/m4/ --include=/yocto/resin-board/build/tmp/work/cortexa9hf-neon-poky-linux-gnueabi/libmbim/1.24.2-r0/recipe-sysroot-native/usr/share/aclocal/ --force
| configure.ac:65: error: Unexpanded AX_ macro found. Please install GNU autoconf-archive
|       If this token and others are legitimate, please use m4_pattern_allow.
|       See the Autoconf documentation.
| configure.ac:66: error: Unexpanded AX_ macro found. Please install GNU autoconf-archive
| ERROR: autoreconf execution failed.
| autoreconf: /yocto/resin-board/build/tmp/work/cortexa9hf-neon-poky-linux-gnueabi/libmbim/1.24.2-r0/recipe-sysroot-native/usr/bin/autoconf failed with exit status: 1
| WARNING: exit code 1 from a shell command.
| ERROR: Function failed: do_configure (log file is located at /yocto/resin-board/build/tmp/work/cortexa9hf-neon-poky-linux-gnueabi/libmbim/1.24.2-r0/temp/log.do_configure.18159)
NOTE: recipe libmbim-1.24.2-r0: task do_configure: Failed
ERROR: Task (/yocto/resin-board/build/../layers/meta-balena/meta-balena-common/recipes-connectivity/libmbim/libmbim_1.24.2.bb:do_configure) failed with exit code '1'
NOTE: recipe linux-technologic-4.9.11-r0: task do_compile: Succeeded
NOTE: Tasks Summary: Attempted 4216 tasks of which 3296 didn't need to be rerun and 1 failed.

Summary: 1 task failed:
  /yocto/resin-board/build/../layers/meta-balena/meta-balena-common/recipes-connectivity/libmbim/libmbim_1.24.2.bb:do_configure
Summary: There were 42 WARNING messages shown.
Summary: There were 2 ERROR messages shown, returning a non-zero exit code.
```